src/jmclient/wallet_rpc.py

```diff
@@ -780,15 +781,24 @@ def directsend(self, request, walletname):
             # all error conditions).
             if not self.coinjoin_state == CJ_NOT_RUNNING:
                 raise ActionNotAllowed()
+
+            old_txfee = jm_single().config.get("POLICY", "tx_fees")
+            if "txfee" in payment_info_json:
+                jm_single().config.set("POLICY", "tx_fees",
+                                       payment_info_json["txfee"])
```

Comment:
This errors for me, and indeed I believe it should: the field in the `payment_info_json` is an integer, not a string, and configparser complains that you must set this field with a string:

```
  File "/home/waxwing/joinmarket-clientserver/src/jmclient/wallet_rpc.py", line 788, in directsend
    jm_single().config.set("POLICY", "tx_fees",
  File "/usr/lib/python3.8/configparser.py", line 1200, in set
    self._validate_value_types(option=option, value=value)
  File "/usr/lib/python3.8/configparser.py", line 1185, in _validate_value_types
    raise TypeError("option values must be strings")
builtins.TypeError: option values must be strings
```

Reply:
I'm confused if @theborakompanioni didn't get this error. I'm probably missing some detail.

Reply:
And to complete the testing result: if I change my json in my request to `"txfee": "12000"` instead of `"txfee": 12000`, then, as expected, the code runs correctly.

Reply:
Indeed, I tested with strings!